openapi/workflow_execution_service.swagger.yaml

```diff
@@ -142,12 +166,42 @@ paths:
           description: An unexpected error occurred.
           schema:
             $ref: '#/definitions/ErrorResponse'
+      consumes:
+         - multipart/form-data
       parameters:
-        - name: body
-          in: body
-          required: true
-          schema:
-            $ref: '#/definitions/WorkflowRunRequest'
+        - in: formData
+          name: workflow_params
+          type: string
+          format: application/json
+
+        - in: formData
+          name: workflow_type
+          type: string
+
+        - in: formData
+          name: workflow_type_version
+          type: string
+
+        - in: formData
```

Comment:
Isn't it possible to send in a request body with multipart upload items? We might leave the WorkflowRunRequest untouched and offer the file upload as an option, as opposed to requiring sending strings to multipart file uploads.

Reply:
If I understand you correctly we already discussed that as an option and voted on this path